foreman_maintain.gemspec

```diff
@@ -23,6 +23,7 @@ the Foreman/Satellite up and running."
 
   s.add_dependency 'clamp'
   s.add_dependency 'highline'
+  s.add_dependency 'rexml'
```

Comment:
This is only needed with Ruby 3.0 and it looks like it breaks Ruby 2.0 since it's updating it to a too-new version. I think if you remove this, it should pass on Ruby 2
```suggestion
```

Reply:
oh i totally missed that rexml was included in the base ruby!

Reply:
thanks! this is now green.  I swear i had googled for built in xml parsers prior and didn't get any hits.  Nokigiri has the SEO

Reply:
The only reason I know rexml exists is because Ruby 3.0 no longer ships it by default ;)